src/components/AppHeader/AppHeader.tsx

```diff
@@ -9,15 +12,63 @@ import { MenuItems } from './MenuItems';
 import { MobileMenu } from './MobileMenu';
 import { Notifications } from './Notifications';
 import { PlatformSwitcher } from './PlatformSwitcher';
+import { AccountsInfoLoader } from './SkeletonLoader';
 import './AppHeader.scss';
 
 // TODO: handle local storage values not updating after changing local storage values
 const AppHeader = () => {
     const { isDesktop } = useDevice();
+    const isEndpointPage = getCurrentRoute() === 'endpoint';
     const { activeLoginid, logout } = useAuthData();
+    const { data: activeAccount } = api.account.useActiveAccount();
     const { localize } = useTranslations();
     const oauthUrl = getOauthUrl();
 
+    const renderAccountSection = () => {
+        if (!isEndpointPage && !activeAccount) {
+            console.log('here:', isEndpointPage, activeAccount);
```

Comment:
is the console required?

Reply:
good catch. removed it. thanks @nada-deriv